guides/common/modules/con_troubleshooting-dnf-modules.adoc

```diff
@@ -0,0 +1,50 @@
+[id="Troubleshooting_dnf_Modules{context}"]
+= Troubleshooting DNF Modules
+
+If the module fails to enable, it can mean an incorrect module is enabled.
+In that case, you have to resolve dependencies manually as follows.
+List the enabled modules:
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf module list --enabled
+----
+
+== Ruby
```

Comment:
Ideally, this would be in it's own separate file with an anchor. You can then include this "troubleshooting ruby" file directly below the "troubleshooting concept" with a heading level +1.

Reply:
Can we go with `troubleshooting_ruby` and `troubleshooting_postgresql` instead of the current i.e. `troubleshooting_dnf_modules` ?

With your view, I don't feel `postrgresql` alone should be in **troubleshooting_dnf_modules**

Reply:
I will not block this PR if you keep it as is; but it'd be nice if every heading was in its own file.

Yes, ruby is just one example; everything would have to be moved into separate files.